webapp/src/validation/preValidation.ts

```diff
@@ -1,9 +1,40 @@
-export function preValidate(value: any): string[] { 
-  const errorMessages: string[] = []
+import { ValidationError } from "./validationError";
+
+export function preValidate(row: number, columnName: string, value: any): Set<ValidationError> { 
+  const validationErrors = new Set<ValidationError>()
 
   if (parseInt(value, 10) === 1337) {
-    errorMessages.push("Oops! That value is too cool to be true!");
+    validationErrors.add({
+      index: row,
+      column: columnName,
+      errorMessage: "Oops! That value is too *sick* to be true!"
+    })
+  }
+
+  if (columnName === "DBH"){
+    if (!value) {
+      validationErrors.add({
+        index: row,
+        column: columnName,
+        errorMessage:"Gotta enter a value"
+      })
+    }
+
+    if (parseInt(value, 10) > 200) {
```

Comment:
maybe for the future we could extract these specific values (e.g. 200) into a config instead of hardcoded, but for now its good 👍